vlc/rootfs/etc/services.d/vlc/run

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/with-contenv bashio
 # ==============================================================================
-# Start Z-Wave JS service for Z-Wave radio
+# Start C-VLC daemon and enable discovery
 # ==============================================================================
-readonly PASSWORD="$(cat /data/secret)"
+readonly TELNET_PASSWORD="$(cat /data/telnet-secret)"
+readonly INGRESS_PASSWORD="$(cat /data/ingress-secret)"
 
 # Send out discovery information to Home Assistant
 ./discovery &
```

Comment:
Need also update the discovery data

Reply:
Yep, I missed that. Thanks.

Reply:
Resolved with https://github.com/home-assistant/addons/pull/1878/commits/94f57fc15c33cf41a0a5c212906208797cde01f0